app/models/merchant.rb

```diff
@@ -20,11 +20,34 @@ def self.most_items(amount = nil)
     .limit(amount)
   end
 
-  def self.revenue(date = Date.today.to_s)
+  def self.revenue(date = nil)
```

Comment:
This was not covered correctly in the Model testing. This method will also probably more closely resemble the show method below after refactoring.